drgn_tools/md.py

```diff
@@ -96,6 +97,26 @@ def md_pv_pending_io(rdev: Object) -> str:
         return "-"
 
 
+def md_pv_badblocks(prog: Program, mddev: Object) -> int:
+    num = 0
+    for rdev in for_each_md_pv(prog, mddev):
+        num += rdev.badblocks.count.value_()
+    return num
+
+
+def show_md_pv_badblocks(prog: Program, mddev: Object) -> None:
+    print("%10s: %-8s %-20s" % ("Badblocks", "PV", "[(off,len,ack)]"))
+    for rdev in for_each_md_pv(prog, mddev):
+        if rdev.badblocks.count.value_() == 0:
+            continue
+        print(
+            "%10s: %-8s " % (" ", md_pv_name(rdev)),
+            end="",
+        )
+        bb_list = [(o, l, a) for o, l, a in for_each_badblocks(rdev.badblocks)]
```

Comment:
A tiny stylistic suggestion:
```suggestion
        bb_list = list(for_each_badblocks(rdev.badblocks))
```
But I don't think it's worth going through another round of code review for it. Just FYI for next time :)